packages/block-library/src/post-comments-form/edit.js

```diff
@@ -55,8 +55,12 @@ export default function PostCommentsFormEdit( {
 
 				{ 'open' !== commentStatus && (
 					<Warning>
-						{ __(
-							'Post Comments Form block: comments to this post are not allowed.'
+						{ sprintf(
+							/* translators: 1: Post type (i.e. "post", "page") */
+							__(
+								'Post Comments Form block: comments to this %s are not allowed.'
```

Comment:
@amustaque97 Isn't this also confusing? First, it mentions Post, then it mentions post type? Confusing to me anyway. Is that the actual name of the Block?

Reply:
@alexstine, yes it is the actual name of the blog. You can read here as well. https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/post-comments-form/block.json#L5

`Post Comments Form` is like this block is saying `some error message`... 

WDYT?

Reply:
@amustaque97 One last question, should "block" be "Block"? Not sure of the capitalization requirements but would be good to keep it consistent.

Reply:
Hey @alexstine Alex. I believe we use "block" and not "Block". I will post in the core editor channel and see what others say. I believe we are going for a more sentence kind of structure.

Reply:
https://developer.wordpress.org/block-editor/explanations/user-interface/block-design/#block-identification
https://github.com/WordPress/gutenberg/issues/16118

Looks like I wasn't the only one.

Reply:
Good fint Alex!